wpeverest/everest-forms
# Tweak - Clone or duplicate core form settings email

### All Submissions:




### Changes proposed in this Pull Request:
Previously, on Form Setting -> Email there was only option to add a new email. Now, We have added the feature to duplicate the existing email and its options.

Closes # .

### How to test the changes in this Pull Request:

1. Go to Form Setting-> Email
2. Click on the duplicate icon just beside the delete icon
3. Enter the name for a new email to be duplicated
4. Make changes if you want to have the email content to be modified or updated
5. Click on the Save button
6. You can enable or disable the toggle to send the email to the admin about the user details on user registration
7. Register a new user and check whether the email with the correct email contents and details

### Types of changes:


### Other information:


### Changelog entry

> Tweak - Clone or duplicate core form settings email
